source/mainnet/smart-contracts/tutorials/voting/voting-dapp.rst

```diff
@@ -2,29 +2,31 @@
 .. _voting-frontend:
 
 ========================
-Setting up the front end
+Setting up the frontend
 ========================
 
-In this part, you will create a web front end. Users can interact with the smart contract more easily
-using a front end compared to interacting with the node directly.
-Non-tech users might find it inconvenient to interact with smart contracts via the Concordium node
+This is the second part of :ref:`the voting smart contract tutorial<voting-sc>`. In this part of the tutorial,
+we will create a web frontend application to make it easier for users to interact with the smart contract.
+
+Non-tech users will likely not be able to interact with smart contracts via a Concordium node
 and some users may choose not to host their own Concordium node locally. This tutorial part shows you
 a setup that eliminates the need for the user to host their own Concordium node.
 
-You can lower the bar for entry by coding an appealing front end that provides additional information
-to your potential customers. Web front ends are a familiar sight nowadays, but to use
-the front end, users will also need to download a web wallet as a browser extension.
-The installation and safe usage of the |bw| might be new for people using your front end.
+You can lower the bar for entry by coding an appealing frontend that provides additional information
+to your users. Web frontends are a familiar sight nowadays, but to use
+the frontend, users will also need to download the |bw| (or simply "the browser wallet") as a browser extension.
+The installation and safe usage of the browser wallet might be new for people using your frontend.
+
 Providing comprehensive explanations and step-by-step guides on your website on topics
-related to the |bw| is important for a good user experience. The |bw|
+related to the browser wallet is important for a good user experience. The browser wallet
```

Comment:
I agree with you that the naming `browser wallet` is much more natural than `|bw| = Wallet for Web` and I did not support the naming decision in the past. We need to get a consensus in the company to rename it throughout the documentation since in the past people agreed and pushed for calling it `Wallet for Web`.

As such, we should change the |bw| to `browser wallet` (if agreed internally) instead of partially using the constants and plain `browser wallet` in the text. The current mix will make it hard to have consistent naming throughout the documentation.

Reply:
My biggest complaint is that `|bw|` expands to `Concordium Wallet for Web` which is just _super long_ and really only makes sense when referring to the wallet in a title or maybe the first time you mention it. I think having `|bw|` expand to that is okay, but it makes no sense to use that expansion everywhere in the prose, as it makes the writing very unnatural.

I don't even know if using such expansions provide a big benefit, finding stuff by a simple search or regex is fine I think? But yea we should agree on how we refer to these things.

Reply:
To be clear, I am not advocating for changing the official name of the wallet or anything. I am using "browser wallet" like a pronoun or a shorthand to refer to "Concordium Wallet for Web".

I personally find that "web wallet" is a bit of an inaccurate name though, as it is not a web application, but a browser extension.